packages/superset-ui-demo/storybook/stories/plugins/legacy-preset-chart-big-number/BigNumber/BigNumberStories.tsx

```diff
@@ -74,7 +74,10 @@ export const nullInTheMiddle = () => (
     width={400}
     height={400}
     queryData={{ data: withNulls(testData, 3) }}
-    formData={formData}
+    formData={{
+      ...formData,
+      timeGrainSqla: 'P1W', // weekly
```

Comment:
do you want to make another story for this or remove the change?

Reply:
Added a new story.